src/features/currentTodo.ts

```diff
@@ -6,5 +6,10 @@ const initialState = null as Todo | null;
 export const currentTodoSlice = createSlice({
   name: 'currentTodo',
   initialState,
-  reducers: {},
+  reducers: {
+    viewTodo: (_todos, action) => action.payload,
```

Comment:
The `viewTodo` reducer updates the state with the payload from the action. Ensure that this reducer is correctly dispatched with the appropriate payload (a `Todo` object or `null`) to update the current todo state as expected.